test/integration/cli-domain-package-server-script/cli-domain-package-server-script.js

```diff
@@ -399,7 +399,10 @@ describe('cli : domain : package-server-script', function(){
               'for(var i=1e12;;){ y = 546; }' +
               'do { z = 342; } while(true);' +
               '};' +
-              'const sayHello = (name) => `Hello ${name}`;' +
+              'const sayHello = (name) => {' +
+              '  if(name!=="John Doe") infiniteLoops();' +
+              '  return `Hello ${name}`;' +
+              '};' +
```

Comment:
I had to trick babili and its treeshaking as the previous test was just removing the infiniteLoops() function as it had no calls (and therefore not making the infinite loops replacements).